e2e_samples/parking_sensors_synapse/scripts/verify_prerequisites.sh

```diff
@@ -3,7 +3,7 @@
 # Check if required utilities are installed
 command -v jq >/dev/null 2>&1 || { echo >&2 "I require jq but it's not installed. See https://stedolan.github.io/jq/.  Aborting."; exit 1; }
 command -v az >/dev/null 2>&1 || { echo >&2 "I require azure cli but it's not installed. See https://bit.ly/2Gc8IsS. Aborting."; exit 1; }
-command -v makepasswd >/dev/null 2>&1 || { echo >&2 "I require makepasswd. See https://bit.ly/3GneVxD. Aborting."; exit 1; }
+# command -v makepasswd >/dev/null 2>&1 || { echo >&2 "I require makepasswd. See https://bit.ly/3GneVxD. Aborting."; exit 1; }
```

Comment:
"makepasswd" is not available on Mac at all (https://discussions.apple.com/thread/2303658).
Either we can use some other common utility or write our own function and remove this dependency. Because of this check, the script is failing on Mac even if we provide the password as part of env. variable. That's why I commented it for now. Even if this check isn't there, the script anyway fail in the very beginning so I though removing it might be a good enough option.